src/toast/observation_data.py

```diff
@@ -535,6 +529,18 @@ def _accel_delete(self):
         elif use_accel_jax:
             self._data = accel_data_delete(self._data)
 
+    def _accel_reset(self):
+        log = Logger.get()
+        log.verbose(f"DetectorData _accel_reset")
+        if use_accel_omp:
+            accel_data_reset(self._raw)
+        elif use_accel_jax:
+            # NOTE: we create an array of zeroes directly on GPU to avoid
```

Comment:
this could become a call to accel_reset once it is updated, i will do it in a later PR

Reply:
Ok, I left this for the next PR :-)